MiniSim/Menu.swift

```diff
@@ -34,19 +34,16 @@ class Menu: NSMenu {
     }
     
     func getDevices() {
-        DispatchQueue.global(qos: .userInitiated).async {
```

Comment:
This code was the root cause of everything being not in the main queue. And that's the most important change in the PR. New `DeviceService.getAllDevices` executes callback closure on the main queue by default. This allows us to remove any `DispatchQueue.main` calls in the `Menu`.

Reply:
Okay, great thanks for the explanation!